code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -305,11 +305,9 @@
 
 /obj/machinery/smartfridge/black_box/Initialize()
 	. = ..()
-	var/static/obj/machinery/smartfridge/black_box/current
-	if(current && current != src)
-		qdel(src, force=TRUE)
-		return
-	current = src
+	if(GLOB.blackbox != src)
+		return INITIALIZE_HINT_QDEL_FORCE
```

Comment:
Ok, we like, definitely don't need this. I have no clue what the `force` is forcing there, but I can't think of anything.

Reply:
look at it's destroy

Reply:
honestly I'm up for straight up removing the thing, but I think that should be done somewhere else

Reply:
this is so fucking stupid bro but fair enough